File: ExtendedDataGridView Control/[C#]-ExtendedDataGridView Control/C#/ExtendedDatagridViewCS/Printing/yC.cs
namespace ExtendedDataGridView
{
    public class yC
	{

		private int _Y;

		private int _Height;
		public yC(int y, int h)
		{
			this.Y = y;
			this.Height = h;
		}

		public int Y {
			get { return _Y; }
			set { _Y = value; }
		}

		public int Height {
			get { return _Height; }
			set { _Height = value; }
		}

	}
}

File: ExtendedDataGridView Control/[VB]-ExtendedDataGridView Control/VB/ExtendedDataGridView/ExtendedDataGridView/Printing/yC.vb
Public Class yC

    Private _Y As Integer
    Private _Height As Integer

    Public Sub New(y As Integer, h As Integer)
        Me.Y = y
        Me.Height = h
    End Sub

    Public Property Y As Integer
        Get
            Return _Y
        End Get
        Set(value As Integer)
            _Y = value
        End Set
    End Property

    Public Property Height As Integer
        Get
            Return _Height
        End Get
        Set(value As Integer)
            _Height = value
        End Set
    End Property

End Class
